### PR TITLE
[4.3] Remove message on login

### PR DESCRIPTION
**Pull request by obuisard**
This is an alternative to PR #40531 as an answer to issue #40458.

### Summary of Changes

A quick solution seems to be the removal of the message when users log in. 
It does not completely revert PR #30834, as messaging users when they log out does add feedback to users they did not have before.
The log out message can easily be adapted via the use of a language override.

### Testing Instructions

Login and logout of a website in the frontend.

### Actual result BEFORE applying this Pull Request

You have a message when logging in and a message when logging out.

### Expected result AFTER applying this Pull Request

There is no longer a message when the user logs in except for the initial feedback users used to get in prior versions.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by chmst** [comment]
Looks good for me.

File: language/en-GB/com_users.ini [hunk 1]
```diff
@@ -30,7 +30,6 @@ COM_USERS_FIELD_RESET_CONFIRM_USERNAME_LABEL="Username"
 COM_USERS_FIELD_RESET_PASSWORD1_LABEL="Password"
 COM_USERS_FIELD_RESET_PASSWORD1_MESSAGE="The passwords you entered do not match. Please enter your desired password in the password field and confirm your entry by entering it in the confirm password field."
 COM_USERS_FIELD_RESET_PASSWORD2_LABEL="Confirm Password"
-COM_USERS_FRONTEND_LOGIN_SUCCESS="You have been logged in."
```
Comment (brianteeman) [inline language/en-GB/com_users.ini:33]: technically you can't delete the string but just mark it as deprecated and remove in5

Comment (obuisard) [reply language/en-GB/com_users.ini:33]: You are right, will change that.

**Comment by LiamLafferty1** [comment]
I also found out that I had to " Remove message on login ", on this free dating app Stars Align https://starsalign.co/, how to resolve the issue?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40539">issues.joomla.org/tracker/joomla-cms/40539</a>.</sub>

**Comment by brianteeman** [comment]
spam

**Comment by hans2103** [comment]
I have tested this item :white_check_mark: successfully on 32dad37e14f67ec0648ceec1029c292e833fb509<br /><br />Tested successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40539">issues.joomla.org/tracker/joomla-cms/40539</a>.</sub>

**Comment by Jurre279** [comment]
I have tested this item :white_check_mark: successfully on 32dad37e14f67ec0648ceec1029c292e833fb509<br /><br />Tested this patch.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40539">issues.joomla.org/tracker/joomla-cms/40539</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40539">issues.joomla.org/tracker/joomla-cms/40539</a>.</sub>

**Comment by sdwjoomla** [comment]
Thanks Olivier

**Comment by joomleb** [comment]
Hi guys,
in the same way would be useful to add finally the Registration form redirection setting #37566

**Pull request merged**